### PR TITLE
Add conversation mirage setup and remove un-needed mirage models

### PR DESCRIPTION
**Pull request by snewcomer**
Getting something showing in 4200 land.  

Another note/question is if we want mirage to auto-detect the model, thus removing the need to define a mirage model (got in version 0.3.2 I think).  Lmk if that is something you want me to remove if the mirage model isn't needed.

### CODE REVIEW

File: config/environment.js [hunk 1]
```diff
@@ -173,6 +173,7 @@ module.exports = function(environment) {
     ENV['simple-auth'] = {
       store: 'simple-auth-session-store:ephemeral'
     };
+    ENV.SOCKETS_BASE_URL = '';
```
Comment (joshsmith) [inline config/environment.js]: Could we add this between API/WEB?

File: mirage/factories/conversation-part.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    return moment().subtract(i, 'days');
+  },
+  readAt(i) {
+    return moment().subtract(i, 'days');
+  },
+  updatedAt(i) {
+    return moment().subtract(i, 'days');
+  }
+  // author: belongsTo('user', { async: true }),
+  // conversation: belongsTo('conversation', { async: true })
```
Comment (joshsmith) [inline mirage/factories/conversation-part.js]: You can remove these two lines.

File: mirage/models/conversation.js [hunk 3]
```diff
@@ -1,7 +0,0 @@
-import { Model, belongsTo, hasMany } from 'ember-cli-mirage';
-
-export default Model.extend({
-  message: belongsTo(),
-  user: belongsTo(),
-  conversationParts: hasMany()
-});
```
Comment (joshsmith) [inline mirage/models/conversation.js]: Why is this being removed?

Comment (snewcomer) [reply mirage/models/conversation.js]: https://github.com/samselikoff/ember-cli-mirage/pull/1056

So mirage can auto generate mirage models based off of the ember-data model, avoiding mismatches b/w the two.

Lmk what you think?  We can keep slowly removing these as needed or all of them in a separate PR.

Comment (joshsmith) [reply mirage/models/conversation.js]: Ah okay thanks. Maybe open a separate issue so others are tracking that’s the case as a point of reference?

**Comment by joshsmith** [comment]
@snewcomer want to squash and merge when you're ready?

**Pull request merged**